### ISSUE TITLE
No option to set Primary in video stack

### ISSUE THREAD

**Issue by inthreedee** [photoprism/photoprism#1030]
When videos are stacked together, for example an original and a trimmed version, the "Primary" button does not appear to change which of the two versions will play when selected in your library.

**Reproduction steps:**
1. Upload a video with an original and an edited version.
2. Make sure photoprism has correctly stacked them during indexing.
3. Select the video in photoprism and click the edit button, then click on the "Files" tab.
4. "Unstack" is available, but there is no option to set the edited or original file as Primary.

**version:**
 210211-b9595dd4-Linux-aarch64

**Comment by lastzero** [comment]
Yes, we don't support stacks of videos yet. The original issue was "simple video support".

**Comment by joachimtingvold** [comment]
The "lack" of setting primary video is compounded with the "Place & Time" stacking enabled. A new video recorded straight after another (i.e. stopping+starting quickly to create new video) currently triggers stacked video, but no way to select primary. I'm not sure if all videos gets encoded regardless of their primary status? If they do, it should be fairly feasible to change the primary video?
